### DISCUSSION TITLE
how to get full av.open(...., options) options's document?

### DISCUSSION THREAD

**Discussion by makerjackie** [PyAV-Org/PyAV#1232] [1. Help]
I know I can search https://ffmpeg.org/ffmpeg.html or use `ffmpeg --help` to find options, 

for example
```
> ffmpeg --help
....
Video options:
-r rate             set frame rate (Hz value, fraction or abbreviation)
-s size             set frame size (WxH or abbreviation)
```

but we use `video_size` instead of `s` in code, How do I know the mapping relationship between them?
```
 options={
            'video_size': '640x480'
        })
```

**Comment by xiaofanku** [comment]
i wanI also want to know. I tried width, height, and vf, but none of them worked
